src/client/app/components/BaseLayout/BaseLayoutHeader.tsx

```diff
@@ -233,7 +240,7 @@ const BaseLayoutHeader = ({
             headers.map(
               (header) =>
                 (header.public ? !isLoggedIn : isLoggedIn) &&
-                !header.hidden && (
```

Comment:
can i check what was the context behind this removal?

Reply:
Headers' objects does not have 'hidden' field, so this was not used